default/scripting/species/common/native_fortification.macros

```diff
@@ -0,0 +1,33 @@
+// param1: detection
+// param2: defense
+// param3: shields
+// param4: troops
+//currently unused
+NATIVE_FORTIFICATION_CUSTOM
+'''
+    [[NATIVE_PLANETARY_DETECTION(@1@)]]
+    [[NATIVE_PLANETARY_DEFENSE(@2@)]]
+    [[NATIVE_PLANETARY_SHIELDS(@3@)]]
+    [[NATIVE_PLANETARY_TROOPS(@4@)]]
+'''
+
+//currently unused
+NATIVE_FORTIFICATION_HIGH
+'''[[NATIVE_FORTIFICATION_CUSTOM(30,30,30,30)]]'''
+
+//currently unused
+NATIVE_FORTIFICATION_MEDIUM
+'''[[NATIVE_FORTIFICATION_CUSTOM(20,20,20,20)]]'''
+
+//currently unused
+NATIVE_FORTIFICATION_LOW
+'''[[NATIVE_FORTIFICATION_CUSTOM(10,10,10,10)]]'''
+
+//currently unused
+NATIVE_FORTIFICATION_MINIMAL
+'''[[NATIVE_PLANETARY_SHIELDS(1)]]'''
```

Comment:
1 troops would seem more sensible than 2 shield as the minimum...

Reply:
Well, native colonies all (currently, afaik) already have troops.  They have a default shield value of 0.

Adding a shield value of 1 would force the player to bring a combat ship to the planet they want to invade, instead of being able to use only troop ships.

That is my thinking at least.

Reply:
more so that a minimally defensive planet wouldn't likely have a shield set up, but it would sensibly have a token garrison of troops.

Reply:
> more so that a minimally defensive planet wouldn't likely have a shield set up, but it would sensibly have a token garrison of troops.

I think this makes sense in terms of 'fluff', but it's not clear to me that adding a single troop has any qualitative effect on how difficult it is to penetrate a planet's defenses (at most it's one more troop ship to bring along).

I would expect this case to be covered by simply omitting any Native_Fortification (or implementing an empty macro NATIVE_FORTIFICATION_NONE), and relying on the population's 'default' troops (i.e. those provided by [[GREAT_DEFENSE_TROOPS]] / [[AVERAGE_DEFENSE_TROOPS]] / etc).

(Aside: Fluffwise i envision these Native_Fortification defenses as factions loyal to their homeworld but who lack interest in planetary defense after being subsumed by an empire.  This explains why the bonus disappears when the planet becomes "owned".)

Reply:
In that case, I'd add a weak attack rather than weak shields.

Reply:
> In that case, I'd add a weak attack rather than weak shields.

Again, i don't think it's clear that really would do anything significant to oppose invasion.  A planet with an attack value and no shield can be directly invaded (InvadeOrder::Check in util/Order.cpp checks only that planet's shield value is zero, not planet's attack value), so (presuming it's a small attack value of say, 1) the invasion force could still consist of only troop ships.

Also note that the shield value of 1 is what all empire planets have as default defense (from the starting tech DEF_ROOT_DEFENSE in https://github.com/freeorion/freeorion/blob/master/default/scripting/techs/defense/Defense.focs.txt ).